language/Korean/strings.po

```diff
@@ -1925,42 +1955,42 @@ msgstr "되감기"
 
 #: src/frontend/mame/ui/tapectrl.cpp:109
 msgid "Fast Forward"
-msgstr "빨리감기"
+msgstr "빨리 감기"
 
 #: src/frontend/mame/ui/inputopts.cpp:75
 msgctxt "menu-inputopts"
 msgid "Input Settings"
-msgstr ""
+msgstr "입력 설정"
 
 #: src/frontend/mame/ui/inputopts.cpp:97
 msgctxt "menu-inputopts"
 msgid "Input Assignments (this system)"
-msgstr ""
+msgstr "입력 할당(현 시스템)"
 
 #: src/frontend/mame/ui/inputopts.cpp:99
 msgctxt "menu-inputopts"
 msgid "Analog Input Adjustments"
-msgstr ""
+msgstr "아날로그 입력 설정"
 
 #: src/frontend/mame/ui/inputopts.cpp:101
 msgctxt "menu-inputopts"
 msgid "Keyboard Selection"
-msgstr ""
+msgstr "키보드 선택"
 
 #: src/frontend/mame/ui/inputopts.cpp:103
 msgctxt "menu-inputopts"
 msgid "Toggle Inputs"
-msgstr ""
+msgstr "입력 전환"
```

Comment:
This is the third place in these menus where you’ve used a different term for toggle inputs.